src/mp/sys/fnmp.vcxproj

```diff
@@ -66,4 +71,4 @@
     </Link>
   </ItemDefinitionGroup>
   <Import Project="$(UndockedDir)vs\windows.undocked.targets" />
-</Project>
+</Project>
```

Comment:
nit: add a trailing empty newline

Reply:
For all these files (VS deletes them for some reason).

Reply:
Will do.
Just to know - if VS really wants to remove them, why do we care about keeping them?